collections/crowdsecurity/supabase-compose.md

```diff
@@ -0,0 +1,24 @@
+## Supabase Docker Compose collection
+
```

Comment:
Could you add a link to supabase docker compose?

Reply:
done via https://github.com/crowdsecurity/hub/pull/689/commits/952132ed7371fc8ce5976e3b6e4fa46b55ddad3b